### PR TITLE
fix(plugin-edusharing): use open-iframe-resizer

### PR DESCRIPTION
**Pull request by LarsTheGlidingSquirrel**

### CODE REVIEW

**Review by hejtful — APPROVED** [review]
Looks good!

For the follow-up, I changed the `plugin-injection` to `open-iframe-resizer` in [this PR](https://github.com/serlo/frontend/pull/4315), and it looks good! Although I'm not very familiar with `plugin-injection`, so I might have missed something.

Please feel free to merge the PR and also remove `iframe-resizer` from editor dependencies. Or we can also just merge this PR and wait for Botho to approve the `plugin-injection` change. Your call :)

**Comment by elbotho** [comment]
@hejtful your PR changes `serlo-injection` (not `injection`) and this is only used outside of serlo.org so I trust your and @LarsTheGlidingSquirrel judgement and tests here :)

File: apps/web/src/pages/content-only/[...slug].tsx [hunk 1]
```diff
@@ -52,7 +52,10 @@ export default renderedPageNoHooks<SlugProps>(({ pageData }) => {
       serloEntityData={{ entityId }}
       authorization={pageData.authorization}
     >
-      <LazyIframeResizer />
+      <Script
+        src="https://cdn.jsdelivr.net/npm/@open-iframe-resizer/core@1.2.1/dist/index.min.js"
```
Comment (elbotho) [inline apps/web/src/pages/content-only/[...slug].tsx:56]: could we host this on serlo.org to avoid any privacy concerns?

Comment (LarsTheGlidingSquirrel) [reply apps/web/src/pages/content-only/[...slug].tsx:56]: I think so. Just have not done it yet.

**Pull request merged**